Sources/Core/AWSClientRuntime/Sources/AWSClientRuntime/Plugins/DefaultAWSClientPlugin.swift

```diff
@@ -15,10 +15,13 @@ public class DefaultAWSClientPlugin: Plugin {
     }
```

Comment:
Now that defaults provider for retry strategy options actually takes in user-exposed config now, let's use em.